code/modules/mob/living/simple_animal/abnormality/he/you_strong.dm

```diff
@@ -162,9 +162,6 @@
 	if(old_part.type in list(/obj/item/bodypart/r_leg/grown_strong, /obj/item/bodypart/l_leg/grown_strong, /obj/item/bodypart/r_arm/grown_strong, /obj/item/bodypart/l_arm/grown_strong))
 		to_chat(M, "<span class='notice'>Only original parts are accepted.</span>")
 		return
-	if(alert(M, "Would you like to put your [old_part.name] into [src]?", "You Must Become Stronger", "Yes", "No") != "Yes")
```

Comment:
Is removing this message intentional and necessary?

Reply:
I believe with the time of the do_after and the text-description, it's clear what occurs. As well as the alert box allowed for WAYYYY to many issues that more or less weren't stoppable without what would probably be shitcode.